apps/css-workshop/src/pages/breadcrumbs.astro

```diff
@@ -38,7 +38,13 @@ import IconButton_ from '../components/IconButton.astro';
       </Button_>
       <ol class='iui-breadcrumbs-list'>
         <li class='iui-breadcrumbs-item'>
-          <a class='iui-anchor iui-breadcrumbs-content' href='#'> Root</a>
+          <a
```

Comment:
These should all use the `Button.astro` component.

Before:
```html
<a class='iui-anchor iui-breadcrumbs-content' href='#'> Root</a>
```

Current:
```html
<a
  class='iui-anchor iui-button-base iui-button iui-field iui-breadcrumbs-content'
  data-iui-variant='borderless'
  href='#'
>
  <span>Root</span>
</a>
```

Expected:
```html
<Button_
  as='a'
  class='iui-anchor iui-breadcrumbs-content'
  variant='borderless'
  href='#'
>Root</Button_>
```

Reply:
Looks like `Button.astro` doesn't support `as='a'` right now.  @Ben-Pusey-Bentley, can you help with this?

Reply:
This might be all you need to support `as='a'`:

<details><summary>Button.astro</summary>

```diff
@@ -5,12 +5,13 @@ type Props = {
   isActive?: boolean;
   dropdown?: boolean;
   idea?: boolean;
+  as?: string;
 } & astroHTML.JSX.ButtonHTMLAttributes;
 
-const { class: className, variant, size, isActive, dropdown, idea, ...props } = Astro.props;
+const { as: Element = 'button', class: className, variant, size, isActive, dropdown, idea, ...props } = Astro.props;
 ---
 
-<button
+<Element
   class:list={[
```

</details>

Reply:
Astro component updated and being used for breadcrumb anchors. https://github.com/iTwin/iTwinUI/pull/1919/commits/84b7e39dcf12ca4a3e16f902897d9f20e785d847